.github/workflows/create_tag.yml

```diff
@@ -130,38 +152,47 @@ jobs:
         id: tag_output
         run: |
           echo "tag_ref=${{steps.create_tag.outputs.tag_ref}}" >> $GITHUB_OUTPUT
+          echo "custom_version=${{steps.create_tag.outputs.custom_version}}" >> $GITHUB_OUTPUT
 
       - name: Set Label
         id: set_label
         shell: bash
         run: |
           if [ -f convoy.yaml ]; then
```

Comment:
Set a variable  (not GITHUB_OUTPUT) `validate_label=false` as first line after `run: |`; Then change that variable to true as needed. Finally at end you can have one line that will do
```
echo "validate_label=$validate_label" >> $GITHUB_OUTPUT
```